src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/util/PaginationUtils.kt

```diff
@@ -8,21 +8,25 @@ import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.SortDirection
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.model.PaginationMetadata
 
 fun getPageable(sortBy: String, sortDirection: SortDirection?, page: Int?): Pageable? {
+  return getPageableWithSize(sortBy, sortDirection, page, 10)
+}
+
+fun getPageableWithSize(sortBy: String, sortDirection: SortDirection?, page: Int?, pageSize: Int): Pageable? {
   return if (page != null) {
     val sort = if (sortDirection == SortDirection.desc) {
       Sort.by(sortBy).descending()
     } else {
       Sort.by(sortBy).ascending()
     }
-    PageRequest.of(page - 1, 10, sort)
+    PageRequest.of(page - 1, pageSize, sort)
```

Comment:
i'm not sure it matters in your context, but I think the metadata will still always return a page size of 10?

Reply:
Good catch, this has been fixed now.